incident-bot.json

```diff
@@ -27,6 +27,18 @@
       "Type" : "AWS::EC2::VPC::Id",
       "Default": ""
     },
+    "Daemon": {
+      "Description" : "Choose which daemon controller you wish to use",
+      "Type" : "String",
+      "Default": "runit",
+      "AllowedValues": ["runit", "supervisor"]
```

Comment:
Original hubot cookbook supported both, I decided to keep both as options. Some people may have a preference on daemon type.